command/action.go

```diff
@@ -30,8 +23,8 @@ func (c *ActionCommand) Help() string {
 	helpText := `
 Usage: nomad action [options] <action>
 
-  Perform a predefined command inside the environment of the given allocation
-  and task.
+  Perform a predefined command inside the environment of a given context.
+  Currently this acts as a wrapper around the job action command.
 
   When ACLs are enabled, this command requires a token with the 'alloc-exec',
   'read-job', and 'list-jobs' capabilities for a task's namespace. If
```

Comment:
I can't comment on the right line because it wasn't modified, but I think I left an `=` in `-group=<group-name>` 😅 

Moreover, should this command include all flags? The `nomad status` command only lists the ones _it_ cares about:
https://github.com/hashicorp/nomad/blob/f66eb83fc035e250569e13d392dab80e1b4cc9a4/command/status.go#L23-L40

Reply:
Yeah good call, keeping the flags on `job action` and ditching them on `action`.